package.json

```diff
@@ -47,7 +47,7 @@
     "react-dom": "15.3.2"
   },
   "devDependencies": {
-    "@kadira/storybook": "2.24.1",
+    "@kadira/storybook": "^2.35.3",
```

Comment:
I was thinking, should we write a utility that validates package.json for semver best practices. My feeling is that deps and devDeps should use exact, but peerDeps should use caret operator. We could create a standalone util for use in JDL but also bake it into js-builder by default. Or maybe something comparable already exists, can look. WDYT @scherler ?